plugins/zendesk/icon_zendesk/connection/connection.py

```diff
@@ -28,3 +28,13 @@ def connect(self, params):
 
         self.client = zenpy.Zenpy(**creds)
         self.logger.info("Connect: Connecting...")
+
+    def test(self):
+        try:
+            self.client.users()
+        except zenpy.lib.exception.APIException as e:
+            self.logger.debug(e)
+            raise PluginException(
+                cause=f"Zendesk API connection test failed.",
+                assistance="Make sure your credentials are correct.",
+            )
```

Comment:
```suggestion
            raise ConnectionTestException(
                cause=f"Connection to Zendesk failed: {e.response}",
                assistance="Make sure your credentials are correct.",
                data=e
            )
```
Also there are some other exceptions we should possibly catch here: http://docs.facetoe.com.au/api_objects.html#exceptions-objects-reference

Reply:
Made a separate ticket to add more specific exceptions: https://issues.corp.rapid7.com/browse/SOAR-8392